src/code.sh

```
@@ -81,150 +53,354 @@ _trim_fastq_endings () {
+       tar -xf /home/dnanexus/in/genome_lib/*.tar.gz -C /home/dnanexus/
+       lib_dir=$(find . -type d -name "*CTAT_lib*")
+
+       # get FusionInspector Docker image by its ID
+       docker load -i /home/dnanexus/in/docker/*.tar.gz
+       DOCKER_IMAGE_ID=$(docker images --format="{{.Repository}} {{.ID}}" | grep "^trinityctat/fusioninspector" | cut -d' ' -f2)
+
+       SECONDS=0
+
+       # set up the FusionInspector command
+       wd="$(pwd)"
+       out_filename=${samplename}_${fusions_name%.*}
```

Shellcheck (0.10.0)

[warning] samplename is referenced but not assigned.

(SC2154)